download-ctags-installer.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+set -x
+# Commit from 2022-03-01. Please always pick a commit from the main branch.
+export SOURCEGRAPH_COMMIT=20497508d57afd4bbd35597629779255d772a7f8
+wget https://raw.githubusercontent.com/sourcegraph/sourcegraph/$SOURCEGRAPH_COMMIT/cmd/symbols/ctags-install-alpine.sh -P $1
+chmod +x "$1/ctags-install-alpine.sh"
```

Comment:
can we just run the script in this script? In both CI and dockerfile after downloading we just run.

Also, please remember to cleanup... IE delete the downloaded script.

If you want to be fancy you can do `wget -O - $url | sh` and avoid writing the script to disk / cleanup

Reply:
`-O -` nice!